admin/create-theme/theme-readme.php

```diff
@@ -4,37 +4,35 @@ class Theme_Readme {
 	/**
 	* Build a readme.txt file for CHILD/GRANDCHILD themes.
 	*/
-	public static function build_readme_txt( $theme ) {
-		$slug                   = $theme['slug'];
-		$name                   = $theme['name'];
-		$description            = $theme['description'];
-		$uri                    = $theme['uri'];
-		$author                 = $theme['author'];
-		$author_uri             = $theme['author_uri'];
-		$copy_year              = gmdate( 'Y' );
-		$wp_version             = get_bloginfo( 'version' );
-		$image_credits          = $theme['image_credits'] ?? '';
-		$is_parent_theme        = $theme['is_parent_theme'] ?? false;
-		$original_theme         = $theme['original_theme'] ?? '';
+	public static function build_readme_txt( $theme, $update_current_theme = false ) {
+		$slug                = $theme['slug'];
+		$name                = $theme['name'];
+		$description         = $theme['description'];
+		$uri                 = $theme['uri'];
+		$author              = $theme['author'];
+		$author_uri          = $theme['author_uri'];
+		$copy_year           = gmdate( 'Y' );
+		$wp_version          = get_bloginfo( 'version' );
+		$image_credits       = $theme['image_credits'] ?? '';
+		$recommended_plugins = $theme['recommended_plugins'] ?? '';
+		$is_parent_theme     = $theme['is_parent_theme'] ?? false;
+		$original_theme      = $theme['original_theme'] ?? '';
+
+		// Handle copyright section.
 		$new_copyright_section  = $is_parent_theme || $original_theme ? true : false;
 		$original_theme_credits = $new_copyright_section ? self::original_theme_credits( $name, $is_parent_theme ) : '';
+		$copyright_section      = self::copyright_section( $new_copyright_section, $original_theme_credits, $name, $copy_year, $author, $image_credits );
 
-		$default_copyright_section = "== Copyright ==
```

Comment:
I've refactored this and moved it lower down so that it's contained in `copyright_section()`.